lib/python/Screens/Setup.py

```diff
@@ -213,9 +213,6 @@ def logIncludeElementError(self, element, type, token):
 	def selectionChanged(self):
 		if self["config"]:
 			self.setFootnote(None)
-			self["description"].text = self.getCurrentDescription()
-		else:
```

Comment:
Please don't remove the else.

Reply:
The "else" is junk that should not be displayed in the GUI to users. If you don't want the commit please don't merge it.